samples/documentation/lulesh_sample1_unix_commented.yaml

```diff
@@ -187,10 +191,41 @@ study:
       # on 'run-lulesh_*' which declares dependency on ALL combinations of
       # 'run-lulesh').
 
+####################################
+# Parameter Block
+####################################
+# The parameter block contains all the things we'd like to vary in the study.
+# Currently, there are two modes of operating in the specification:
+# 1. If a parameter block is specified, the study is expanded and considered a

```

Comment:
Does the parameter also have to be referenced in a step?

Reply:
I added a statement to this that a parameter does not necessarily need to be used.

Reply:
Is the question if global.parameters implies the non-Linear expansion, even if the parameters are unused?